dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/main/java/org/apache/dolphinscheduler/plugin/alert/email/EmailConstants.java

```diff
@@ -18,6 +18,8 @@
 package org.apache.dolphinscheduler.plugin.alert.email;
 
 public final class EmailConstants {
+    public static final String LINE_SEPARATOR = System.getProperty("line.separator");
```

Comment:
I don't think there is necessity to add a constant, just use `System.lineSeparator()`

Reply:
> 

ok ,fine